code/obj/submachine/robotics.dm

```diff
@@ -315,14 +315,27 @@
 	//can be obj/machinery/light for wall tubes, obj/machinery/light/small for wall bulbs. Either mode does floor fittings because there's only one type of those
 	var/dispensing_fitting = /obj/machinery/light
 	var/list/setting_context_actions
+	var/list/common_actions
+	var/list/page_1_actions
+	var/list/page_2_actions
 	contextLayout = new /datum/contextLayout/experimentalcircle
```

Comment:
pass in a value to have this circle embiggened (32 is the default size; the ai hologram menu also does this)
```suggestion
	contextLayout = new /datum/contextLayout/experimentalcircle(36)
```

it's going to look a bit weird with the circles have a different count of options, but this will help it not overlap as much:
![Screenshot 2024-06-29 120236](https://github.com/goonstation/goonstation/assets/91498627/cbb2d405-34e6-4eac-bbe6-eb2737ff0f72)
![Screenshot 2024-06-29 120241](https://github.com/goonstation/goonstation/assets/91498627/2e52c593-b52d-4fd0-801a-e9e44f9b1b53)

you could, alternatively, have the P1/P2 buttons change the circle size but that might be weirder to use in practice